src/content/docs/merge-queue/parallel-checks.mdx

```diff
@@ -133,6 +133,36 @@ Parallel Checks adapt automatically as your code and rules evolve:
 
 ## Important Considerations
 
+### In‑place checks (no drafts)
+
+Mergify runs checks directly on the original pull request (instead of using
+temporary draft PRs) only when ALL of the following are true:
```

Comment:
Does this also needs to mention bot_account?

Reply:
Good idea, i added a note, tell me what you think  about it